app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,13 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+export const calculateCreatedAgo = date => {
+  dayjs.extend(relativeTime);
+  const createdAgo = dayjs(date).fromNow();
+  return createdAgo;
+};
+
+export const formatDatetoWeekdayTime = date => {
+  const weekdayTime = dayjs(date).format("dddd, hh:mmA");
+  return weekdayTime;
```

Comment:
The extra variable `weekdayTime` is unnecessary over here.

Reply:
Removed